dotnet/src/AutoGen.SourceGenerator/Template/FunctionCallTemplate.tt

```diff
@@ -81,7 +81,7 @@ namespace <#=NameSpace#>
                         Name = @"<#=parameter.Name#>",
 <#}#>
 <#if (parameter.Description != null) {#>
-                        Description = @"<#=parameter.Description#>",
+                        Description = @"<#= parameter.Description.Replace("\"", "\"\"") #>",
```

Comment:
The `FunctionCallTemplate.cs` also needs to be updated as well.

`.tt` is a t4 template which is used to generate the corresponding .cs file. Please refer to the below link for further information

https://learn.microsoft.com/en-us/visualstudio/modeling/code-generation-and-t4-text-templates?view=vs-2022

Reply:
Also, can you add a test case in `AutoGen.SourceGenerator.Tests` to cover this change?

Reply:
> The `FunctionCallTemplate.cs` also needs to be updated as well.
> 
> `.tt` is a t4 template which is used to generate the corresponding .cs file. Please refer to the below link for further information
> 
> https://learn.microsoft.com/en-us/visualstudio/modeling/code-generation-and-t4-text-templates?view=vs-2022

Hey @LittleLittleCloud , I tried to generate the corresponding cs file using the mono.TextTransform but I am hit with the `Processing 'FunctionCallTemplate.tt' failed.
/var/folders/rf/y8w5vbhx46bg1lg5181s1hyw0000gn/T/tmpEqtGsr.tmpdir/GeneratedTextTransformation.cs(22,20): ERROR: The type or namespace name 'SourceGeneratorFunctionContract' could not be found (are you missing a using directive or an assembly reference?)`. Do you have any fix on how I can generate the cs file using this t4? Thanks!

Reply:
> > The `FunctionCallTemplate.cs` also needs to be updated as well.
> > `.tt` is a t4 template which is used to generate the corresponding .cs file. Please refer to the below link for further information
> > https://learn.microsoft.com/en-us/visualstudio/modeling/code-generation-and-t4-text-templates?view=vs-2022
> 
> Hey @LittleLittleCloud , I tried to generate the corresponding cs file using the mono.TextTransform but I am hit with the `Processing 'FunctionCallTemplate.tt' failed. /var/folders/rf/y8w5vbhx46bg1lg5181s1hyw0000gn/T/tmpEqtGsr.tmpdir/GeneratedTextTransformation.cs(22,20): ERROR: The type or namespace name 'SourceGeneratorFunctionContract' could not be found (are you missing a using directive or an assembly reference?)`. Do you have any fix on how I can generate the cs file using this t4? Thanks!

Solved it, Thanks no need to address this!